crates/dbsp/src/circuit/runtime.rs

```diff
@@ -467,17 +432,38 @@ impl Runtime {
         WORKER_INDEX.get()
     }
 
-    /// Returns the minimum number of bytes in a batch to spill it to
-    /// storage. For threads that run without a runtime, this method returns
-    /// `usize::MAX`.
-    pub fn min_storage_bytes() -> usize {
+    /// Returns the minimum number of bytes in a batch to spill it to storage,
+    /// or `None` if this thread doesn't have a [Runtime] or if it doesn't have
+    /// storage configured.
+    pub fn min_storage_bytes() -> Option<usize> {
         RUNTIME.with(|rt| {
-            rt.borrow()
-                .as_ref()
-                .map_or(usize::MAX, |runtime| runtime.0.min_storage_bytes)
+            Some(
+                rt.borrow()
```

Comment:
It seems crazy we have to write all this to access a setting (no action needed just an observation)

Reply:
Yeah, it's completely wild, isn't it?